assets/js/components/adminbar/AdminBarUniqueVisitors.js

```diff
@@ -72,15 +75,15 @@ const AdminBarUniqueVisitors = ( { WidgetReportZero, WidgetReportError } ) => {
 		] )
 	);
 
-	if ( ! hasFinishedResolution ) {
-		return <PreviewBlock width="auto" height="59px" />;
-	}
-
 	if ( error ) {
 		return <WidgetReportError moduleSlug="analytics" error={ error } />;
 	}
 
-	if ( isZeroReport( analyticsData ) ) {
+	if ( ! hasFinishedResolution || isGatheringData === undefined ) {
+		return <PreviewBlock width="auto" height="59px" />;
+	}
```

Comment:
Same as above.

Reply:
@felixarntz For this one, the error rendering logic stayed where it was. Do you think we should move it above the loading logic to be consistent with how the other widgets are?

Reply:
I think this is looking good as is.